content/docs/user-guide/interactivity/robotics/concepts-and-components-overview.md

```diff
@@ -0,0 +1,107 @@
+
+Typically, you will be creating publishers and subscriptions. This is done through [rclcpp API](https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1Node.html#classrclcpp_1_1Node). Example:
+
+```
+auto ros2Node = ROS2Interface::Get()->GetNode();
+AZStd::string fullTopic = ROS2Names::GetNamespacedName(GetNamespace(), m_MyTopic);
+m_myPublisher = ros2Node->create_publisher<sensor_msgs::msg::PointCloud2>(fullTopic.data(), QoS());
+```
+
+Note that QoS class is a simple wrapper to [`rclcpp::QoS`](https://docs.ros.org/en/humble/p/rclcpp/generated/classrclcpp_1_1QoS.html).
+
+## Components overview
+
+- __Central Singleton__
+  - `ROS2SystemComponent`
+- __Core abstractions__
+  - `ROS2FrameComponent`
+  - `ROS2SensorComponent`
+- __Sensors__
+  - `ROS2CameraSensorComponent`
+  - `ROS2GNSSSensorComponent`
+  - `ROS2IMUSensorComponent`
+  - `ROS2LidarSensorComponent`
+  - `ROS2OdometrySensorComponent`
+- __Robot control__
+  - `AckermannControlComponent`
+  - `RigidBodyTwistControlComponent`
+  - `SkidSteeringControlComponent`
+- __Spawner__
+  - `ROS2SpawnerComponent`
+  - `ROS2SpawnPointComponent`
+- __Vehicle dynamics__
+  - `AckermannVehicleModelComponent`
+  - `SkidSteeringModelComponent`
+  - `WheelControllerComponent`
+- __Robot Import (URDF) system component__
+  - `ROS2RobotImporterSystemComponent`
+
+See the [class diagram](/docs/user-guide/interactivity/robotics/class-diagram/) to understand how components are connected.
+
+### Frames
+
+`ROS2FrameComponent` is a representation of an interesting physical part of the robot. It handles the spatio-temporal relationship between this part and other frames of reference. It also encapsulates namespaces, which help to distinguish between different robots and different parts of the robot, such as in the case of multiple identical sensors on one robot.
+
+All Sensors and the Robot Control component require `ROS2FrameComponent`.
+
+### Sensors
+
+Sensors are components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish it to ROS 2 domain.
+
+- Each sensor has a configuration, including one or more Publishers.
+- Sensors publish at a given rate (frequency).
+- Some sensors can be visualized.
+
+If you intend to add your own sensor, it might be useful to look at how sensors already provided within the O3DE ROS2 Gem are implemented.
+
+### Robot Control
+
+The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through:
+
+- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
+- [AckermannDrive](https://github.com/ros-drivers/ackermann_msgs/blob/master/msg/AckermannDrive.msg)
+  The component subscribes to these command messages on a configured topic. The topic is `cmd_vel` by default, in a namespace as dictated by __ROS2Frame__.
```

Comment:
- It sounds like the sentence "The component subscribes..." applies to both Twist and AckermannDrive, but it's currently rendererd together on the website. I added some space to separate them. 
- Nit: Don't need a period for short-phrased lists
```suggestion
- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages
- [AckermannDrive](https://github.com/ros-drivers/ackermann_msgs/blob/master/msg/AckermannDrive.msg)

The component subscribes to these command messages on a configured topic. The topic is `cmd_vel` by default, in a namespace as dictated by __ROS2Frame__.
```